mysql-test/include/mtr_check.sql

```diff
@@ -34,6 +34,7 @@ BEGIN
      AND variable_name != 'INNODB_USE_NATIVE_AIO'
      AND variable_name not like 'GTID%POS'
      AND variable_name != 'GTID_BINLOG_STATE'
+     AND variable_name != 'AUTO_INCREMENT_INCREMENT'
```

Comment:
`auto_increment_increment` is something that affects the file format of the entire data node. If we ignore changes to it, there is a huge risk of logical data corruption. Can this be fixed in the `galera_3nodes` tests themselves?

Reply:
fixed